include/zephyr/bluetooth/classic/hfp_hf.h

```diff
@@ -168,16 +427,520 @@ struct bt_hfp_hf_cb {
  */
 int bt_hfp_hf_register(struct bt_hfp_hf_cb *cb);
 
-/** @brief Handsfree client Send AT
+/** @brief Initiate the service level connection establishment procedure
+ *
+ *  Initiate the service level connection establishment procedure on the
+ *  ACL connection specified by the parameter `conn` using the specific
+ *  RFCOMM channel discovered by the function `bt_br_discovery_start`.
+ *
+ *  The parameter `hf` is a output parameter. When the service level
+ *  connection establishment procedure is performed without any error,
```

Comment:
Should this say "initiated" instead of "performed"? If I understood correctly, the procedure is not necessarily complete when the function returns, however "performed" could be interpreted as "completed".

Reply:
Updated. Changed it to "initiated".